lib/FrontendTool/FrontendTool.cpp

```diff
@@ -1,4 +1,3 @@
-//===--- FrontendTool.cpp - Swift Compiler Frontend -----------------------===//
```

Comment:
Looks like this line got accidentally deleted somehow. Oops. 😅

Reply:
^ Could you please fix this one + squash + force-push? We can run tests after that, hopefully no more changes needed.🤞

Reply:
ok pushed it.